### ISSUE TITLE
Multiple elements at the same line

### ISSUE THREAD

**Issue by raboof** [bramp/js-sequence-diagrams#130]
Right now each new element (arrow or note) is drawn at a new 'line'.

It might be neat to also allow multiple elements on one line - though it'll probably be up to the user to make sure they don't cause overlap.

This could be represented using indentation:

```
A->B: Broadcast trigger
B->A: Broadcast message
  B->C: Broadcast message
  Note right of C: Yay!
B->A: Delivery ack
```

```
  A                     B                     C
  |                     |                     |
  |  Broadcast trigger  |                     |
  | ------------------> |                     |
  |                     |                     |
  |  Broadcast message  |  Broadcast message  |
  | <------------------ | ------------------> | Yay!
  |                     |                     |
  |     Delivery ack    |                     |
  | <------------------ |                     |
  |                     |                     |
```

**Comment by raboof** [comment]
```
A->B: Trigger broadcast
B->A: Broadcast message
=B->C: Broadcast message
=Note right of C: Yay
B->A: Ack broadcast
```

![selection_033](https://cloud.githubusercontent.com/assets/131856/9479733/d962bf08-4b7e-11e5-850f-9e36affea73b.png)

**Comment by zagor** [comment]
Yes, I would love this too. I need to describe a system with broadcast messages, and have been unable to find ANY sequance diagram drawing tool that support it.

Just a minor comment about the prototype: There needs to be a marker on the vertical line of the sender, to show who sent the message.

**Comment by raboof** [comment]
You mean this diagram is ambiguous because it could also be interpreted to mean A and C sent 'broadcast' messages to each other? I suppose that makes sense - but might be something for a separate PR/issue. #129 would be an alternative way to remove this ambiguity.

**Comment by zagor** [comment]
I mean if there was a fourth actor D, and B sent to A and D, there would be no visual indication that B sent the message rather than C.

**Comment by raboof** [comment]
Ah yes, that's a more realistic example of the same ambiguity indeed :).

**Comment by ensonic** [comment]
+1 I was looking for  a way to have concurrent notes. At least those are not ambigous.

Maybe the start of a line can have a small dot.
